app/Http/Controllers/Api/UserController.php

```diff
@@ -63,6 +63,27 @@ public function update(Request $request, int $id)
         return $user;
     }
 
+    /**
+     * Give credits to a user.
+     *
+     * @param Request $request
+     * @param int $id
+     * @return User
+     */
+    public function addCredits(Request $request, int $id)
+    {
+        $discordUser = DiscordUser::find($id);
+        $user = $discordUser ? $discordUser->user : User::findOrFail($id);
+
+        $request->validate([
+            "credits"      => "required|numeric|min:0|max:1000000",
+        ]);
+
+        $user->increment('credits', $request->credits);
```

Comment:
Make sure to check if the users credits doesn't go over the max amount of credits before adding it. If it does go over the max ammount this wil result in a 500 sql error.

Example check pulled from voucher controller (redeem function)
```
if ($request->user()->credits + $voucher->credits >= 99999999) throw ValidationException::withMessages([
            'code' => "You can't redeem this voucher because you would exceed the credit limit"
        ]);
```